k-distribution/include/kframework/ktest.mak

```diff
@@ -45,6 +45,9 @@ SOURCE_EXT?=$(or $(and $(wildcard $(DEF).k), k), $(or $(and $(wildcard $(DEF).md
 ifeq ($(KOMPILE_BACKEND),llvm)
 KRUN=$(abspath $(MAKEFILE_PATH)/../../bin/kx)
 endif
+ifeq ($(KOMPILE_BACKEND),haskell)
+KRUN=$(abspath $(MAKEFILE_PATH)/../../bin/kx)
+endif
```

Comment:
Are we testing the old `krun` codepath at all at this point then?

Reply:
We still are falling back to the old tools if we don't have direct generated parsers, so it should be OK.

Reply:
Well, we are still testing the code path via the java backend in places...